src/main/webapp/dev/js/common/instructorFeedbackResults.es6

```diff
@@ -319,6 +319,12 @@ function prepareInstructorFeedbackResultsPage() {
         });
     };
 
+    // decrease top-margin of expand button, if status message is present
+    if ($('#statusMessagesToUser').html().search('div') > 0) {
+        $('#expand-collapse-button').css('margin-top', '5px');
+        $('#expand-collapse-button + br').remove();
+    }
```

Comment:
@whipermr5 I added some top margin to the status message div and this is how it looks like now. Do let me know if this looks good. This does not use any jquery (pure inline CSS)
![1](https://user-images.githubusercontent.com/10449636/27985323-7b8b3f8e-6406-11e7-9356-3bfe281f4db0.JPG)
![2](https://user-images.githubusercontent.com/10449636/27985324-7b8d6e62-6406-11e7-985a-a8fcae04d9ea.JPG)